cli_reference/openshift_developer_cli/odo-release-notes.adoc

```diff
@@ -5,10 +5,96 @@ include::modules/common-attributes.adoc[]
 :context: odo-release-notes
 toc::[]
 
-include::modules/openshift-developer-cli-odo-notable-features.adoc[leveloffset=+1]
+[id="odo-notable-improvements_{context}"]
+== Notable improvements in {odo-title}{nbsp}{odo-ver}
 
-include::modules/openshift-developer-cli-odo-getting-support.adoc[leveloffset=+1]
+* IBM Z and PowerPC architecture binaries are now available.
+* `odo catalog` has been improved to provide more useful output information.
+* Information prompts have been added to the `odo service create` and `odo url create` commands, providing possible next step scenarios.
```

Comment:
It seems pr https://github.com/openshift/odo/pull/2346 is responsible for ```Information prompts for `odo service create and providing possible next step scenarios``` not for ```odo url create commands```. If yes then the proper sentence would be ```Information prompts have been added to the `odo service create` and providing possible next step scenarios.```

@cdrage can you please confirm this ?

Reply:
Yes, you're correct. Sorry! Remove the URL portion please. Only `service create` was updated.